astro.config.ts

```diff
@@ -60,7 +61,10 @@ export default defineConfig({
 		astroCodeSnippets(),
 	],
 	vite: {
-		plugins: [vitePreact()],
```

Comment:
I'm actually not sure why `@preact/preset-vite` is added here, but `pnpm dev` and HMR seems to still work well after removing this.

Reply:
For some reason I needed this to get `preact/compat` working when we first migrated to it (the search component we depend on is a React component), looks like the search is still working after removing so I guess we’ve fixed whatever this was patching back then 🥳